content/id/docs/tasks/configure-pod-container/share-process-namespace.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Pembagian Namespace Process yang dijalankan pada Kontainer-Kontainer pada Pod yang sama
+min-kubernetes-server-version: v1.10
+content_type: task
+weight: 160
+---
+
+<!-- overview -->
+
+{{< feature-state state="stable" for_k8s_version="v1.17" >}}
+
+Dokumen ini akan menjelaskan menkanisme konfigurasi pembagian namespace
```

Comment:
```suggestion
Dokumen ini akan menjelaskan menkanisme konfigurasi pembagian namespace
```

Reply:
typo not fixed yet?
Also, "Namespace" is an API name, do we need to translate? if no need, we change all "namespace" into "Namespace".
```suggestion
Dokumen ini akan menjelaskan mekanisme konfigurasi pembagian Namespace
```

Reply:
I think the namespace here is referring to linux "process namespace" https://en.wikipedia.org/wiki/Linux_namespaces#Process_ID_(pid)

not the Namespace API object, it's better to have it italicized